lib/stringify.js

```diff
@@ -49,6 +50,10 @@ var stringify = function stringify( // eslint-disable-line func-name-matching
         obj = filter(prefix, obj);
     } else if (obj instanceof Date) {
         obj = serializeDate(obj);
+    } else if (generateArrayPrefix === 'comma' && Array.isArray(obj)) {
+        obj = obj.map(function (val) {
+            return typeof val === 'object' ? JSON.stringify(val) : val;
```

Comment:
why `JSON.stringify`? what if `val` also has an array, or also has a Date object, etc? I think this needs to recurse here.

Reply:
this is why I wrote 'there should be more common acknowledge to discuss about it'

Reply:
I delete support for nested object.
`JSON.stringify` is not good enough, though I do not think further recurse would be appropriate. I don't see if it is a good idea to take all into account.